### DISCUSSION TITLE
Shop Create3 link broken?

### DISCUSSION THREAD

**Discussion by slowrunner** [iRobotEducation/create3_docs#618] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Chrome and Safari

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

x.x

### Which RMW is your robot running?

I don't know

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

The [Shop Create3 link](https://edu.irobot.com/shop/coding-robots/create?variant=269697) reports "too many redirects"

<img width="1574" alt="Screenshot 2024-11-02 at 5 38 59 PM" src="https://github.com/user-attachments/assets/319560fa-27ec-4674-8df8-1b75d29f9bd0">

**Comment by alsora** [comment]
Hi @slowrunner , sorry about this.
We are in the process of migrating the website.

I'll follow up here as soon as the process is completed and the links should be functional again.

**Comment by slowrunner** [comment, marked as answer]
Guessing this is the end of iRobot Education after the Nov 5th 2024 restructuring.  It appears the Create3 as a stand-alone product product has been withdrawn from both direct and vendor sales.  ClearPathRobotics stated they have acquired enough Create3 to continue TurtleBot 4 sales.

This is very sad end for a great product and developer team.
